models/item.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+const ItemSchema = mongoose.Schema({
+  user_uid: {
+    type: String,
+    ref: 'users'
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  expiry: {
```

Comment:
maybe we should add expired as an additional property to do a quick check if the item is expired or not, so in the future we can filter the list by active or inactive items.

Reply:
Agreed, the models were just setup as a placeholder really and something to test, we can update these with all the document fields we require.

Reply:
Was thinking the same. Then we can run a service/whatever at 12:05am ish and set expired items to isExpired or similar, remove from search and then let the user deal with them.

Reply:
Should it not be "expiry_date" as per https://souperheroes.atlassian.net/wiki/spaces/AP/pages/68550667/MongoDB+Data+Structure ?